frontend/src/views/producer/dashboard/event-explore/index.js

```diff
@@ -1,31 +1,79 @@
 import React from 'react';
 import styled from 'styled-components';
 import { render } from 'react-dom';
+import EventCard from '../../../../components/event-card';
+import Button from '../../../../components/button';
+import Typography from '../../../../components/typography';
+import { connect } from '../../../../utils';
 
 const Wrapper = styled.div`
+  padding: 1rem;
+  padding-top: calc(3rem + 5rem);
   height: 100%;
-  width: 22rem;
+  width: 25rem;
   background-color: white;
   box-shadow: 3px 0 12px rgba(0, 0, 0, 0.25);
   position: absolute;
   left: 0;
   top: 0;
   z-index: 10;
+  overflow: auto;
 `;
 const WrapperFlat = styled(Wrapper)`
   position: relative;
+  pointer-events: none;
   background-color: transparent;
   box-shadow: none;
-  margin-right: 2rem;
+  flex-shrink: 0;
+  flex-grow: 0;
 `;
 
-export default class EventExplorer extends React.Component {
+const EmptyStateContainer = styled.div`
+  padding: 2rem;
+  background-color: rgba(0, 0, 0, 0.2);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+class EventExplorer extends React.Component {
   render() {
+    const { events } = this.props.store;
+    console.log(this.props.store.events.toJSON(), this.props.store.isEmpty);
     return (
       <React.Fragment>
-        <Wrapper />
         <WrapperFlat />
+        <Wrapper>
+          {!this.props.store.isEmpty ? (
+            Object.values(this.props.store.events.toJSON()).map(event => (
+              <EventCard
+                style={{ marginBottom: '0.5rem' }}
+                key={event.id}
```

Comment:
How about just passing the event to the EventCard component as event={event} to reduce the amount of props and mapping needed here?

Reply:
sure, i will fix this in my next pr